utils/src/main/kotlin/Utils.kt

```diff
@@ -567,3 +571,51 @@ fun Throwable.showStackTrace() {
     // https://discuss.kotlinlang.org/t/if-operator-in-function-expression/7227.
     if (printStackTrace) printStackTrace()
 }
+
+
+/**
+ * Return Closable which temporarily moves the given directories to a temporary directory on initialization. On close
+ * it move them back to their original location overwriting any conflicting existing directory.
+ */
+fun stashDirectories(vararg directories: File) : Closeable {
+    val directorySet = HashSet(Arrays.asList(*directories))
```

Comment:
setOf(*directories)

Reply:
Actually, you should make that whole thing a one-liner:

    fun stashDirectories(vararg directories: File): Closeable = StashDirectory(setOf(*directories))